libqtile/widget/pomodoro.py

```diff
@@ -171,4 +171,4 @@ def _send_notification(self, urgent, message):
         send_notification("Pomodoro", message, urgent=urgent)
 
     def poll(self):
-        return self.fmt.format(self._get_text())
+        return self.display_format.format(self._get_text())
```

Comment:
Would this work without changing behaviour or configuration options?:
```suggestion
        self._text = self._get_text()
        return self.formatted_text
```
And dropping the redundant `fmt` option altogether from the child class.

Reply:
Yes - I realised this late last night (but only after I'd submitted this).